seed/populate-database.ts

```diff
@@ -0,0 +1,16 @@
+import * as dotenv from 'dotenv';
+import { Seed } from './seed';
+import { setup } from '../src/setup';
+import { User } from '../src/entity/User';
+
+dotenv.config({ path: `${process.cwd()}/test.env` });
+
+async function populateDatabase(generateUsersQtt: number) {
```

Comment:
```suggestion
async function populateDatabase(usersQuantity: number) {
```

Não é legal abreviar coisas nas nomenclaturas de variáveis. É um risco a ser tomado em relação a manutenção que a meu ver é meio desnecessário, dado que são apenas alguns caracteres a mais.

Reply:
Perfeito, segui sua sugestão, corrigido.